### PR TITLE
move JS out of haml into JS pipeline

### PR DESCRIPTION
**Pull request by Bjvanminnen**
This change is motivated by some future changes I want to make (i.e. adding some React to this haml view). Decided to do this as two separate PRs as the change to move this JS out of haml has value independent of future changes.

### CODE REVIEW

File: apps/src/code-studio/teacher.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+function resizeScrollable() {
+  var newHeight = $('.teacher-panel').innerHeight() -
+      $('.teacher-panel h3').outerHeight() -
+      15 - // magic..
+      $('.non-scrollable-wrapper').outerHeight();
+  $('.scrollable-wrapper').css('max-height', newHeight);
+}
+
+$(document).ready(() => {
+  $(window).resize(dashboard.utils.debounce(resizeScrollable, 250));
```
Comment (Bjvanminnen) [inline apps/src/code-studio/teacher.js]: Moving this into JS means that all of our usages of dashboard.utils are now in the JS pipeline. The only things in utils are `debounce` and `throttles`, and both just map to lodash methods. We could, at this point, replace all usages with lodash directly. However, doing so in this particular case would mean adding all of lodash to this bundle. Could be lodash is small enough that we don't really care about the duplicated download?

Comment (pcardune) [reply apps/src/code-studio/teacher.js]: I'm pretty sure you can import individual lodash functions, so you don't have to bring in all of lodash.

**Comment by pcardune** [comment]
LGTM

File: apps/src/code-studio/teacher.js (hunk 1 above)
Comment (Bjvanminnen) [reply apps/src/code-studio/teacher.js]: You can! Sweet.

**Comment by Bjvanminnen** [comment]
I just realized as my next step I need access to the reduxStore used by code-studio, which means this has to be part of that bundle :(

**Comment by Bjvanminnen** [comment]
Enough has changed here that it probably deserves a second code review. @pcardune could you take another look?

My next step has me writing some React for teacher.js that connects to our reduxStore from code-studio. In the current world, that means we need to be part of the code-studio bundle. As such, teacher no longer gets its own bundle

**Comment by pcardune** [comment]
LGTM

**Pull request merged**